### ISSUE TITLE
Implement the regularizer described in "Regularizing RNNs by Stabilizing Activations"

### ISSUE THREAD

**Issue by bryandeng** [fchollet/keras#2025]
In the ICLR 2016 paper [Regularizing RNNs by Stabilizing Activations](http://arxiv.org/abs/1511.08400), the authors propose a simple yet effective regularizer for RNNs, called "norm-stabilizer", which outperforms weight noise and dropout.

Do you think it's a good idea to have this in keras?

**Comment by PiranjaF** [comment]
I think it would be a great idea. It should only be a few lines of code and seems to allow the RNN/LSTM to better generalize beyond the length of the training sequences. Would you look into implementing it?
